src/main/java/core/basesyntax/JavaStreamApi.java

```diff
@@ -29,16 +34,24 @@ public Long elementCount(List<String> elements, String element) {
      * Вернуть Optional первого элемента коллекции</p>
      **/
     public Optional<String> firstElement(List<String> elements) {
-        return null;
+        Optional<String> firstElement = elements.stream().findFirst();
```

Comment:
@SviatoslavExpert Feel free to return the value immediately
```suggestion
        return elements.stream().findFirst();
```